compose/cli/main.py

```diff
@@ -275,6 +276,7 @@ def pull(self, project, options):
 
         project.pull(
             service_names=options['SERVICE'],
+            skip_missing=True if options.get('--skip-missing') else False
```

Comment:
You don't need an if/else here, just `skip_missing=options.get('--skip-missing')`